examples/custom_env.py

```diff
@@ -29,7 +29,7 @@ def __init__(self):
         device = os.environ["CUDA_VISIBLE_DEVICES"]
         logger.info("initializing model on device=%s", device)
 
-    def forward(self, _: dict) -> dict:
+    def forward(self, data: dict) -> dict:
         device = os.environ["CUDA_VISIBLE_DEVICES"]
```

Comment:
```suggestion
        del data
        device = os.environ["CUDA_VISIBLE_DEVICES"]
```
Just to avoid potential lint check error. [ref](https://pycodequ.al/docs/pylint-messages/w0613-unused-argument.html#how-to-fix).

Reply:
Some discussions here: https://github.com/microsoft/pylance-release/issues/194.

This won't be marked as a lint error. Previously I thought we could use the prefix `_` but it conflicts with private attributes in Python.